linked_list/linkedList.js

```diff
@@ -0,0 +1,263 @@
+class Node {
+	// Constructs Node
+	constructor (data) {
+		this.data = data;
+		this.next = null;
+	}
+}
+
+class LinkedList {
+	// Initialises Linked List
+	constructor (size = 0) {
+		this.head = null;
+		this._size = size;
+	}
+
+	// Inserts Element with value at index
+	add (value, index) {
+		if (index < 0 || index > this._size) {
```

Comment:
Suppose (max) size of LinkedList is 5, and a user wants to insert an element at index 5, then they should get an error, right? Since indices go from 0 to 4.